fe1-web/store/reducers/WalletReducer.ts

```diff
@@ -0,0 +1,47 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+/**
+ * This file represents the reducer for the wallet state. Its
+ * job is to handle actions between the application and the
+ * storage (e.g. storing the wallet seed)
```

Comment:
I'm not sure that "Its job is to handle actions between the application and the storage" is very explicit 😅 
You might just want to skip ahead to "Its job is to store the wallet seed".